cypress/e2e/MovieListPage.cy.ts

```diff
@@ -0,0 +1,54 @@
+describe('Movie List Page', () => {
+    beforeEach(() => {
+        cy.visit('http://localhost:3000')
+    })
+
+    it('successfully loads', () => {
+        cy.url().should('include', '/')
+    })
+
+    it('genre select works correctly', () => {
```

Comment:
This is not an end-to-end test, just a unit test checking if the button gets the class after clicking. E2E test should also check other results of the clicking such as loading the movies of a specific genre, or changing the current URL.